apps/meteor/client/components/Omnichannel/CustomFieldsForm/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactElement } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { CustomSelectInput } from './CustomSelectInput';
+import { CustomTextInput } from './CustomTextInput';
+import { useCustomFields } from './hooks/useCustomFields';
+
+const CustomFieldsForm = (): ReactElement | null => {
+	const result = useCustomFields();
+
+	const { register, control } = useFormContext<{ livechatData: Record<string, unknown> }>();
+
+	const { data, isLoading } = result;
```

Comment:
You should deconstruct directly on the hook call. No need for the var result.

Reply:
ok